lms/views/content_item_selection.py

```diff
@@ -8,6 +13,30 @@
 from lms.views.decorators import create_h_user
 
 
+def _check_params(lti_params):
+    except_msg = 'Required LTI data param for content item selection missing: '
```

Comment:
How about using the translation (`_`) on the `except_msg` instead of on each of the individual messages? It actually makes more sense this way as you'd never actually translate the name of the parameters...if that makes sense?